src/VisualStudio/IntegrationTest/IntegrationTests/Extensions/ErrorListExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Common;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Extensions.ErrorList
+{
+    public static partial class ErrorListExtensions
+    {
+        public static void ShowErrorList(this AbstractIntegrationTest test)
+        {
+            test.WaitForAsyncOperations(FeatureAttribute.SolutionCrawler);
+            test.WaitForAsyncOperations(FeatureAttribute.DiagnosticService);
+            test.WaitForAsyncOperations(FeatureAttribute.ErrorSquiggles);
+            test.WaitForAsyncOperations(FeatureAttribute.ErrorList);
+            test.VisualStudio.Instance.ErrorList.ShowErrorList();
```

Comment:
Why does the show need a wait? And why aren't the WaitForAyncOperations in the core function?